autograder/cli/courses/assignments/submissions/proxy/submit.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+import autograder.api.courses.assignments.submissions.proxy.submit
+import autograder.assignment
+import autograder.cli.config
+import autograder.util.timestamp
+
+def run(arguments):
+    result = autograder.api.courses.assignments.submissions.proxy.submit.send(arguments,
+            files = arguments.files, exit_on_error = True)
+
+    if (not result['found-user']):
+        print("Proxy user not found.")
+        return 1
+
+    message = result.get('message', '')
+    if ((message is not None) and (message != '')):
+        # Replace any timestamps in the message.
+        message = autograder.util.timestamp.convert_message(message, pretty = True)
+
+        print("--- Message from Autograder ---")
+        print(message)
+        print("-------------------------------")
+
+    if (result['rejected']):
+        print("Submission was rejected by the autograder.")
+        return 2
+
+    if (not result['grading-success']):
+        print("Grading failed.")
+        return 3
+
+    submission = autograder.assignment.GradedAssignment.from_dict(result['result'])
+    print(submission.report())
```

Comment:
Looks like most of this can probably be shared with `autograder/cli/courses/assignments/submissions/submit.py`.